### ISSUE TITLE
Feature Request: Listing & Editing Properties

### ISSUE THREAD

**Issue by Runi-c** [rjmatthews62/MooCoder#2]
One piece of this behavior currently exists in "Edit > Text List" which lets you edit a property, but it would also be really nice to be able to list and dump properties the same way as "Dump" and "Get Verbs" in a separate "Props" tab, which you could double click to edit just like the verb editor.

**Comment by rjmatthews62** [comment]
Good idea, although I'm not sure how I should represent maps... maybe something like formatted json?

**Comment by rjmatthews62** [comment]
I've implemented Property Listing and editing in the Python version of MooCoder:

[MooCoderPy](https://github.com/rjmatthews62/MooCoderPy)
